tooltips.js

```diff
@@ -0,0 +1,27 @@
+
+function setChildVisibility(element, visibility) {
+    const child = element.firstElementChild
+    if (child != null)
+        child.style.visibility = visibility
+}
+
+const clickButton = document.getElementById("click-button")
+clickButton.onclick = () => {
+    setChildVisibility(clickButton, "visible")
+    setTimeout(() => {
+        setChildVisibility(clickButton, "hidden")
+    }, 2000)
+}
+
+function openModal() {
+    var dialog = document.getElementById("modal-dialog")
+    if (dialog == null)
+        dialog = document.createElement("dialog")
+    dialog.id = "modal-dialog"
```

Comment:
You are the first one who did creating the modal on fly. Good job.
But, I want to say that it is not good to add event listeners describing innerHTML.

Do the next steps:
1. create an element
2. set innerHTML
3. create and use function which set events for all elements inside